libmscore/style.cpp

```diff
@@ -94,7 +94,7 @@ static const StyleType styleTypes[] {
       { Sid::lyricsMinDistance,       "lyricsMinDistance",       Spatium(0.0)  },
       { Sid::lyricsLineHeight,        "lyricsLineHeight",        1.0 },
       { Sid::lyricsDashMinLength,     "lyricsDashMinLength",     Spatium(0.4) },
-      { Sid::lyricsDashMaxLength,     "lyricsDashMaxLegth",      Spatium(0.8) },
+      { Sid::lyricsDashMaxLength,     "lyricsDashMaxLength",     Spatium(0.8) },
```

Comment:
I'm unsure who (or whether at all) to deal with scores containing this typo setting.